packages/ui/CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 The following is a curated list of changes in the Enact ui module, newest changes on the top.
 
+## [unreleased]
+
+### Removed
+
+- `ui/GridListImageItem`
+- `ui/IconButton`
+- `ui/Marquee.MarqueeDecorator` config `className`
+- `ui/RadioDecorator`
+- `ui/Scroller.ScrollerBase`
+- `ui/SlotItem`
+- `ui/ToggleItem`
+- `ui/VirtualList.UiVirtualListBase`
```

Comment:
```suggestion
```
As https://github.com/enactjs/enact/pull/3063, I think we don't need to mention ui/VirtualList.UiVirtualListBase in changelog

Reply:
🫡